mappings/net/minecraft/client/input/KeyCombo.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_4324 net/minecraft/client/input/KeyCombo
```

Comment:
This class was already mapped by my Realms PR.

Reply:
Oops, missed that one! Removed.